app/static/src/css/custom.css

```diff
@@ -15,6 +15,9 @@
     justify-content: center;
 }
 
+.govuk-fieldset__legend {
+    font-weight: bold;
+}
```

Comment:
We shouldn't need any custom CSS for this - we should be able to feed through whatever we need into the form/field, I think ... I'll investigate, because it can be a bit gnarly to thread the needle.

Reply:
We can do this in the template to add a bold class to the field when we render it:

```
            {{
              form.file_format(
                params={
                  "fieldset": {
                    "legend": {
                      "text": form.file_format.label.text,
                      "classes": "govuk-!-font-weight-bold"
                    }
                  }
                }
              )
            }}
```

Then we don't need the `render_kw` bit on the form field instance.